controller/ampachecontroller.php

```diff
@@ -501,4 +502,26 @@ protected function get_cover() {
 			return $r;
 		}
 	}
+
+	/**
+	 * this is not a part of ampache specification
+	 * This is an exposed API endpoint.
+	 * Client uses settings#addUserKey instead
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 * @CORS
```

Comment:
> Do you have an idea how to accomplish this?

Without code duplication.

Workaround: Create a second route in the `SettingController` and call in the method code just the original method to not copy the code but only change the authentication.